avocado/plugins/assets.py

```diff
@@ -333,6 +336,73 @@ def configure(self, parser):
                                  long_arg='--hash',
                                  parser=register_subcommand_parser)
 
+        purge_subcommand_parser = subcommands.add_parser(
+                'purge',
+                help='Removes assets cached locally.')
+
+        help_msg = ("Remove based on a size filter (comparison "
+                    "operator + value) in bytes. Ex '>20', '<=200'. "
+                    "Supported operators: ==, <, >, <=, >=.")
+        settings.register_option(section='assets.purge',
+                                 key='size_filter',
+                                 help_msg=help_msg,
+                                 default=None,
+                                 metavar="FILTER",
+                                 key_type=str,
+                                 long_arg='--by-size-filter',
```

Comment:
Why does this option name have a `filter` suffix, while the `--by-days` one doesn't?

Reply:
Filter on days would be strange, IMO. Would be limited to some operators only. `--days-filter==3` for instance, makes not much sense to me.

Reply:
My point was really that `--by-size` may not a `filter` suffix IMO.

Reply:
hum.. not sure. Since we are not passing a "size" but a filter size I would prefer to be explicit here. `--by-size=">=300"` is not so explicit as `--by-size-filter=">300"`, IMO.